shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/model/TableMetaData.java

```diff
@@ -85,6 +90,16 @@ public TableMetaData(final Collection<ColumnMetaData> columnMetaDataList, final
     public ColumnMetaData getColumnMetaData(final int columnIndex) {
         return columns.get(columnNames.get(columnIndex));
     }
+
+    /**
+     * Get index meta data.
+     *
+     * @param indexIndex  index index
+     * @return index meta data
+     */
+    public IndexMetaData getIndexMetaData(final int indexIndex) {
+        return indexes.get(indexNames.get(indexIndex));
+    }
```

Comment:
Please do not change `TableMetaData`, we just load, the usage is not need to change.